src/scheduler/workers/processListsBeforeAndDuringStart/audit.ts

```diff
@@ -1,17 +1,45 @@
+import { ListItemEvent } from "@prisma/client";
 import { prisma } from "scheduler/prismaClient";
-import type { AuditEvent } from "@prisma/client";
 
-export async function findAuditEvents(
-  annualReviewReference: string,
-  auditEvent: AuditEvent,
-  type?: "user" | "list" | "listItem",
-  itemId?: number
-) {
-  type = type ?? "listItem";
+export async function findAllReminderEvents(annualReviewReference: string, itemId?: number) {
+  const andCondition = [
+    {
+      type: ListItemEvent.REMINDER,
+    },
+    {
+      jsonData: {
+        path: ["reference"],
+        equals: annualReviewReference,
+      },
+    },
+    {
+      ...(itemId && {
+        listItemId: itemId,
+      }),
+    },
+  ];
+
+  try {
+    const result = await prisma.event.findMany({
+      take: 1,
+      orderBy: {
+        time: "desc",
+      },
+      where: {
+        AND: andCondition,
+      },
+    });
+    return { result };
+  } catch (e) {
+    const message = `Unable to find audit records: ${e.message}`;
+    return { error: new Error(message) };
+  }
+}
 
+export async function findReminderAuditEvents(annualReviewReference: string, itemId?: number) {
```

Comment:
maybe `findAllReminderAudits`? one is searching on the event table, the other is on the audit table